src/rgw/rgw_reshard.cc

```diff
@@ -496,6 +496,9 @@ int RGWBucketReshardLock::renew(const Clock::time_point& now) {
 
 int RGWBucketReshard::do_reshard(int num_shards,
 				 int max_entries,
+         FaultInjector<std::string_view>& f,
+         std::optional<std::string> inject_error_at,
+         std::optional<std::string> inject_abort_at,
```

Comment:
inject_error_at and inject_abort_at are both unused in rgw_reshard.cc

Reply:
fixed